src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -793,8 +793,62 @@ internal List<INodeLibraryDependencyInfo> NodeLocalDefinitions
             }
         }
 
+        internal List<INodeLibraryDependencyInfo> ExternalFiles
+        {
+            get
+            {
+                var externalFiles = new Dictionary<object, DependencyInfo>();
```

Comment:
just from a reusability perspective, I think it would be better to have the getter be very small and extract this to a method.

Reply:
Yes, I will refactor this and extract it into a method.